.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: go
+
+go:
+  - "1.10.2"
+
+env:
+  global:
+    - LINUX_BINARY_PATH: ${GOPATH}/bin/terraform-provider-rollbar
+    - DARWIN_BINARY_PATH: ${GOPATH}/bin/darwin_amd64/terraform-provider-rollbar
+
+# Install go dep.
+before_install:
+  - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+
+script:
+  - make vet
+  - make test
+
+# Build and zip the binaries.
+before_deploy:
+  - GIT_TAG=$(git describe --tags)
+  - make build
+  - mv ${LINUX_BINARY_PATH} ${LINUX_BINARY_PATH}_v${GIT_TAG}
+  - zip -j ${LINUX_BINARY_PATH}_v${GIT_TAG}_linux_amd64.zip ${LINUX_BINARY_PATH}_v${GIT_TAG}
+  - make build-darwin
+  - mv ${DARWIN_BINARY_PATH} ${DARWIN_BINARY_PATH}_v${GIT_TAG}
+  - zip -j ${DARWIN_BINARY_PATH}_v${GIT_TAG}_darwin_amd64.zip ${DARWIN_BINARY_PATH}_v${GIT_TAG}
+
+# Deploy only when there's a tag on master.
```

Comment:
what is a "tag on master"? if mean if the tag has the identical commit SHA is the current master?

what about: if the tag names looks like version? something like `/v(\d+)\.(\d+)\.(\d+)/` instead?